mappings/net/minecraft/block/TransparentBlock.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_5190 net/minecraft/block/TransparentBlock
```

Comment:
Transparent or translucent?

Reply:
I guess translucent is better

Reply:
Why translucent? Even glass extends this class. Also upstream makes use of `TransparentBlock` too.